server/app/routes/api/v1/va/serializers.js

```diff
@@ -99,8 +99,9 @@ function serializeAmbassadorForAdmin(ambassador) {
   obj["onboarding_completed"] = ambassador.get("onboarding_completed")
   obj["giftcard_completed"] = ambassador.get("giftcard_completed")
   obj["signup_completed"] = ambassador.get("signup_completed")
-  obj["locked"] = isLocked(ambassador);
+  obj["locked"] = isLocked(ambassador)
   obj["payout_provider"] = ambassador.get("payout_provider")
+  obj["admin_bonus"] = ambassador.get("admin_bonus") ? ambassador.get("admin_bonus").toString() : ""
```

Comment:
why serialize this as a string and not a number?

Reply:
it should be a number, fixed